scripts/gendocs.lua

```diff
@@ -0,0 +1,22 @@
+local docgen = require'docgen'
+
+local function gen()
+  local input_files = {
+    './lua/sql.lua',
+    './lua/sql/table.lua',
+    './lua/sql/stmt.lua',
```

Comment:
@tami5 for which files do you want to generate docs?

Also i think most files don't have a brief
```lua
---@brief [[
--- Desc
---@brief ]]
```

Reply:
I am not sure. I haven't touched that table since i started this PR a "year" ago.

I just rebased :rofl: